Source/iop/Iop_McServ.h

```diff
@@ -198,6 +202,11 @@ namespace Iop
 		static const char* m_mcPathPreference[2];
 		std::string m_currentDirectory;
 		CPathFinder m_pathFinder;
+
+		// Keeps track, if the memory card in
+		// a given slot has already been read,
+		// or if it is a newly inserted card.
+		bool m_knownMemoryCards[2];

```

Comment:
Could we use the newly added `MAX_PORTS` enum here?

Reply:
Done